DB.php

```diff
@@ -18,8 +18,90 @@ public static function getInstance()
         }
```

Comment:
try to leave in Db only methods to work with database:
generic methods:
connect();
rawQuery($sql='');
selectData($table, $conditions=['column'=>['sign'=>'=','val'=>'12','type'=>'INT']]);
updateData();
deleteData();
execute($sql);
private function prepareConditions($conditions=[]);